sdk/Sdk.Generators/FunctionMetadataProviderGenerator.Parser.cs

```diff
@@ -608,6 +635,18 @@ private bool IsEventHubsTriggerValid(IParameterSymbol parameterSymbol, TypeSynta
                     }
                 }
 
+                // Check the default value of IsBatched
+                var eventHubsAttr = attribute.AttributeClass;
+                var isBatchedProp = eventHubsAttr!.GetMembers().Where(m => string.Equals(m.Name, Constants.IsBatchedKey, StringComparison.OrdinalIgnoreCase)).SingleOrDefault();
+                AttributeData defaultValAttr = isBatchedProp.GetAttributes().Where(attr => SymbolEqualityComparer.Default.Equals(attr.AttributeClass, Compilation.GetTypeByMetadataName(Constants.DefaultValueType))).SingleOrDefault();
+                var defaultVal = defaultValAttr.ConstructorArguments.SingleOrDefault().Value!.ToString(); // there is only one constructor arg, the default value
```

Comment:
If we are expecting a bool here, can we do something strongly typed instead of string comparisons?